src/tables.jl

```diff
@@ -2,23 +2,33 @@
 
 ## Chains
 
+function _check_columnnames(chn::Chains)
+    colnames = Symbol.(names(chn))
+    for nm in (:iteration, :chain)
+        nm in colnames && error("'$(nm)' is a reserved column name. Rename the parameter.")
+    end
```

Comment:
`Symbol.(names(chn))` always creates a new array of parameter names, even if the parameter names are already symbols. I'd suggest:
```suggestion
    for name in names(chn)
        symname = Symbol(name)
        if symnname === :iteration || symnname === :chains
            error("'$(name)' is a reserved column name. Please rename the parameter.")
        end
    end
```